src/Service/TransactionService.php

```diff
@@ -125,12 +143,15 @@ public function processTransaction($apiPayment)
 
         switch ($apiPayment->resource) {
             case Config::MOLLIE_API_STATUS_PAYMENT:
+                PrestaShopLogger::addLog(__METHOD__ . ' said: Starting to process PAYMENT transaction.', Config::NOTICE);
```

Comment:
LoggerInterface maybe? ;d

Reply:
In other places it is called like that, not worth effort in 1.6 as we don't really maintain it.